hadoop-ssm-project/src/main/java/org/apache/hadoop/ssm/fetcher/NamespaceFetcher.java

```diff
@@ -203,10 +249,33 @@ public void run() {
         FileStatusInternal[] statuses = batch.getFileStatuses();
         if (statuses.length == batch.actualSize()) {
           this.dbAdapter.insertFiles(batch.getFileStatuses());
+          numPersisted += statuses.length;
         } else {
           FileStatusInternal[] actual = new FileStatusInternal[batch.actualSize()];
           System.arraycopy(statuses, 0, actual, 0, batch.actualSize());
           this.dbAdapter.insertFiles(actual);
+          numPersisted += actual.length;
+        }
+
+        if (LOG.isDebugEnabled()) {
+          LOG.debug(batch.actualSize() + " files insert into table 'files'.");
+        }
```

Comment:
Just for performance consideration. The code will be executed for many many times, so it's better to save some cost when possible.